Servers/UI/OJS.Servers.Ui/ClientApp/src/components/submissions/submissions-list/RefreshableSubmissionsList.tsx

```diff
@@ -12,40 +8,12 @@ const RefreshableSubmissionsList = ({
     items,
     selectedSubmission,
     className = '',
-}: IRefreshableSubmissionsListProps) => {
-    const {
-        state: { currentSubmission },
-        actions: { getSubmissionResults },
-    } = useSubmissionsDetails();
-
-    const submissionsReloadBtnClassName = 'submissionReloadBtn';
-
-    const handleReloadClick = useCallback(async () => {
-        if (isNil(currentSubmission)) {
-            return;
-        }
-
-        const { problem: { id: problemId }, isOfficial, user: { id: userId } } = currentSubmission;
-
-        await getSubmissionResults(problemId, isOfficial, userId);
-    }, [ currentSubmission, getSubmissionResults ]);
-
-    return (
-        <>
-            <SubmissionsList
-              items={items}
-              selectedSubmission={selectedSubmission}
-              className={className}
-            />
-
-            <Button
-              onClick={handleReloadClick}
-              text="Reload"
-              type={ButtonType.secondary}
-              className={submissionsReloadBtnClassName}
-            />
-        </>
-    );
-};
+}: IRefreshableSubmissionsListProps) => (
+    <SubmissionsList
+      items={items}
+      selectedSubmission={selectedSubmission}
+      className={className}
+    />
+);
```

Comment:
The refreshable list should handle Reloading logic internal and only expose callbacks or events where different implementations will be able to add